apps/storybook/src/navigation/GlobalNav.stories.tsx

```diff
@@ -0,0 +1,77 @@
+import type { Meta, StoryObj } from "@storybook/react";
+
+import { Avatar, Link, Text } from "@optiaxiom/react";
+import {
+  GlobalNav,
+  GlobalNavAccount,
+  GlobalNavItem,
+} from "@optiaxiom/react/unstable";
+import {
+  IconBinaryTree,
+  IconChartInfographic,
+  IconChartLine,
+  IconFlag2,
+  IconHistory,
+  IconSettings,
+  IconUsers,
+  IconVocabulary,
+} from "@tabler/icons-react";
+
+export default {
+  component: GlobalNav,
+} as Meta<typeof GlobalNav>;
+
+type Story = StoryObj<typeof GlobalNav>;
+
+const Template: Story["render"] = (args) => (
+  <GlobalNav
+    endDecorator={
+      <GlobalNavAccount
+        avatar={
+          <Avatar src="https://images.unsplash.com/photo-1511485977113-f34c92461ad9?ixlib=rb-1.2.1&w=128&h=128&dpr=2&q=80" />
+        }
+      >
+        <Text fontWeight="500">Rhaenyra Targaryen</Text>
+        <Text fontSize="sm">Optimizely</Text>
```

Comment:
let's move name and organization back to props
```suggestion
      <GlobalNavAccount
        avatar={
          <Avatar src="https://images.unsplash.com/photo-1511485977113-f34c92461ad9?ixlib=rb-1.2.1&w=128&h=128&dpr=2&q=80" />
        }
        name="Rhaenyra Targaryen"
        organization="Optimizely"
```